central/notifier/datastore/singleton.go

```diff
@@ -12,7 +17,11 @@ var (
 )
 
 func initialize() {
-	as = New(store.Singleton())
+	if features.PostgresDatastore.Enabled() {
+		as = New(postgres.New(context.TODO(), globaldb.GetPostgres()))
```

Comment:
context.TODO?

Reply:
Looks like it's a standard for all pg stores.